apps/give-base/src/scripts/before_install.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/ec2-user/server
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)